recipes/goofit/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "2.2.2" %}
+{% set name = "goofit" %}
+
+{% set goofit_proc_version = "1.0.0" %}
+{% set goofit_proc_type = "cpu" if cuda_compiler_version|default("None") == "None" else "gpu" %}
+{% set goofit_cuda_supported = ["None", "9.2", "10.0"] %}
+
+package:
+  name: goofit-split
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82884fd3021555da0953776f81199cbea60287f6bca87b0ecdaa5dd212692c67
+  patches:
+    - patches/thrustorder-2.2.2.patch
+  
+build:
+  skip: true  # [win]
+  {% if cuda_compiler_version|default("None") in goofit_cuda_supported %}
+  {% else %}
+  skip: true
+  {% endif %}
+  number: 0
+
+outputs:
+  - name: goofit-proc
+    version: "{{ goofit_proc_version }}"
+    build:
+      number: 0
+      string: "{{ goofit_proc_type }}"
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/goofit-feedstock
+      license: BSD-3-Clause
+      license_family: BSD
+      summary: A meta-package to select CPU or GPU GooFit build.
+  - name: goofit
+    version: {{ version }}
+    build:
+      number: 0
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("cuda") }}  # [linux and cuda_compiler_version != "None"]
+        - cmake
+        - llvm-openmp  # [osx]
+        - libgomp      # [linux and not aarch64]
+      host:
+        - python
+        - scikit-build
+        - pip
+        - llvm-openmp  # [osx]
+        - libgomp      # [linux and not aarch64]
+      run:
+        - python
+        - numpy >=1.11.1
```

Comment:
Are you linking against numpy's C API? If you aren't, this is fine.

Reply:
Nope, it's PyBind11, which does not link to numpy like Cython does.